packages/core/src/configuration.ts

```diff
@@ -152,10 +152,12 @@ export const timeScale: TimeScaleOptions = {
 };
 
 const isFullScreenEnabled =
-	document['fullscreenEnabled'] ||
-	document['webkitFullscreenEnabled'] ||
-	document['mozFullScreenEnabled'] ||
-	document['msFullscreenEnabled'];
+	document && (
```

Comment:
I checked out this branch and built the `core` package.

I copied over the `bundle.js` and used it in the SSR SvelteKit set-up. Unfortunately, I still receive the "document is not defined error."

However, changing it to `typeof document !== 'undefined'` seems to resolve the SSR issue.

```suggestion
	typeof document !== 'undefined' && (
```